template/user.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../bower_components/app-storage/app-localstorage/app-localstorage-document.html">
+
+<dom-module id="papan-user-data">
+  <template>
+    <app-localstorage-document
+      key="papan-user-data"
+      data="{{user}}">
+    </app-localstorage-document>
+  </template>
+  <script>
+    Polymer({
+      is: 'papan-user-data',
+
+      properties: {
+        user: {
+          type: Object,
+          value: function() {
```

Comment:
Of course, but why must it be a function here? Do you need any kind of "builder" function? Why not directly use the json-like object definition to define the value?